doc/conf.py

```diff
@@ -134,7 +120,7 @@
 project = 'Matplotlib'
 copyright = ('2002 - 2012 John Hunter, Darren Dale, Eric Firing, '
              'Michael Droettboom and the Matplotlib development '
-             'team; 2012 - 2016 The Matplotlib development team')
+             'team; 2012 - 2017 The Matplotlib development team')
```

Comment:
maybe overkill but we could always use 

`"...2012 - %s The Matplotlib development team" % day.year`